### ISSUE TITLE
cannot install helm chart on k8s 1.23.3

### ISSUE THREAD

**Issue by masterkain** [reactiveops/charts#736]
### What happened?

`"Helm install failed: unable to build kubernetes objects from release manifest: unable to recognize \"\": no matches for kind \"APIService\" in version \"apiregistration.k8s.io/v1beta ││ 1\""`

### What did you expect to happen?

chart gets installed with dependencies

### How can we reproduce this?

install helm chart on 1.23.3

### version

chart 5.3.0

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Additional context

```
    metrics-server:
      enabled: true
```

**Comment by sudermanjr** [comment]
Can you please share which chart you are trying to install?

**Comment by masterkain** [comment]
sorry I didn't realize there were more charts, I'm installing goldilocks, here's the HelmRelease I'm using (chart works when setting apiservice to false)

```yaml
apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  name: goldilocks
spec:
  releaseName: goldilocks
  chart:
    spec:
      chart: goldilocks
      version: "5.3.0"
      sourceRef:
        kind: HelmRepository
        name: fairwinds-stable
        namespace: flux-system
  interval: 10m0s
  install:
    remediation:
      retries: 4
  upgrade:
    remediation:
      retries: 4
  values:
    vpa:
      enabled: true
      updater:
        enabled: true
      admissionController:
        enabled: true
      recommender:
        extraArgs:
          prometheus-address: |
            http://loki-stack-prometheus-server.loki-stack.svc.cluster.local
          storage: prometheus
    metrics-server:
      enabled: true
      apiService:
        create: false
```

**Comment by lucasreed** [comment]
Ah, looks like we need to update the version of metrics-server chart we have as a dependency. They fix this in chart version 5.8.1: https://github.com/bitnami/charts/commit/6ee90691341442f1f2bb008ee9f8af6744f7d1a4
